.storybook/preview.js

```diff
@@ -61,14 +56,17 @@ const customViewports = {
   },
 };
 
-addParameters({
+export const parameters = {
+  docs: {
+    theme: sageTheme,
+  },
+  layout: "fullscreen",
```

Comment:
Do we need `layout: "fullscreen"` here then`isFullScreen: false` in the options?